lib/ur/usage_report.go

```diff
@@ -328,7 +328,11 @@ func (s *Service) reportData(ctx context.Context, urVersion int, preview bool) (
 }
 
 func (*Service) UptimeS() int {
-	return int(time.Since(StartTime).Seconds())
+	if int(time.Since(StartTime).Seconds()) != int(time.Now().Unix()) {
```

Comment:
This is somewhat fragile. How about something like `if StartTime.Year() < 2023 { return 0 }` since it's impossible for this code written now to see a start time in the past?

Reply:
(we should maybe also filter unrealistic values in the usage report receiver)

Reply:
> (we should maybe also filter unrealistic values in the usage report receiver)

Good point because people may of course still be running older versions that will keep reporting those wrong uptime stats 🙃. I'll see what I can do about it.

Reply:
> This is somewhat fragile (comparison between two `Now()`s (one implicit in `Since`) with some time in between). How about something like `if StartTime.Year() < 2023 { return 0 }` since it's impossible for this code written now to see a start time in the past?

Done! Force-pushed to fix the commit title and message.

I'm not sure how to proceed with filtering on the receiving side though. Do you mean https://github.com/syncthing/syncthing/tree/main/cmd/ursrv? Is there any way to test the code changes done there locally?